dbaas_nfsaas/faas_provider.py

```diff
@@ -9,27 +9,50 @@
 
 class Provider(object):
 
-    def __init__(self, dbaas_api, host_class):
+    def __init__(self, dbaas_api, host_class, group_klass):
```

Comment:
To follow the pattern of code, maybe will be good u put group_class instead group_klass